frontend/src/lib/components/Support/SupportForm.tsx

```diff
@@ -43,6 +43,7 @@ export function SupportForm(): JSX.Element | null {
     const { objectStorageAvailable } = useValues(preflightLogic)
     // the support model can be shown when logged out, file upload is not offered to anonymous users
     const { user } = useValues(userLogic)
+    const supportRequestKind = useSelector(supportLogic.selectors.kind)
```

Comment:
you don't need to do this separately, on line 41 you can just do:

```
const { sendSupportRequest, kind } = useValues(supportLogic)
```

Selectors are values too!

Reply:
oh, it's also just a part of sendSupportRequest it seems, so you don't really need the selector at all?

Reply:
removed the selector